substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/SecuritySubstitutions.java

```diff
@@ -218,7 +218,7 @@ private static Object replaceAccessControlContext(Object obj) {
     @NeverInline("Starting a stack walk in the caller frame")
     protected Class<?>[] getClassContext() {
         final Pointer startSP = readCallerStackPointer();
-        return StackTraceUtils.getClassContext(1, startSP);
+        return StackTraceUtils.getClassContext(0, startSP);
```

Comment:
Why? Or why it was?

Reply:
One of the reasons this benchmark didn't work was that stacktrace was not equal to the one on java, and it was not equal because we are skipping the first frame. (https://github.com/java-native-access/jna/blob/d367f0220be55b5638b3d324473ef0a7e654d83c/src/com/sun/jna/Native.java#L1516 in this line of code benchmark is accessing caller class and it was shifted by one in NI)